tests/testthat/test-apply_theme.R

```diff
@@ -25,15 +25,15 @@
 #' T1.16 A warning when `grid` is anything but a boolean.
 #' T1.17 No error when `bg` is a string.
 #' T1.18 A warning when `grid` is anything but a boolean.
-#' T2. The `add_theme` function applies the specified changes to a `ggplot` object.
+#' T2. The `apply_theme` function applies the specified changes to a `ggplot` object.
 #' T2.1 No error when a `ggplot` plot is provided, but no theme.
 #' T2.2 No error when a `ggplot` plot and a minimal `visR::define_theme` object are provided.
 #' T2.3 No error when a `ggplot` plot and a complex `visR::define_theme` object are provided.
 #' T2.4 A message when a theme not generated through `visR::define_theme` is provided.
 
 # Requirement T1 ---------------------------------------------------------------
 
-context("add_theme - T1. The `define_theme` function returns an object that has a valid shape for usage in the `add_theme` function.")
+testthat::context("apply_theme - T1. The `define_theme` function returns an object that has a valid shape for usage in the `apply_theme` function.")
 
 testthat::test_that("T1.1 No error when no parameters are specified.",{
   
```

Comment:
Here I still dont see what a 'valid' shape is

Reply:
Will improve the wording :+1: